amundsen_application/static/js/utils/tests/index.spec.ts

```diff
@@ -46,3 +47,21 @@ describe('Navigation utils', () => {
     });
   });
 });
+
+
+describe('dateUtils', () => {
+  describe('formatEpochTime', () => {
+    it('formats an epoch time to the default format', () => {
```

Comment:
Can we have some cases that include the time conversion as well since that's something that's important for our application? I assume Jest would output the timezone in `GMT`? I actually don't know and I'm curious if this sort of test would just work as expected.

Reply:
Oh man, I vaguely remember adding some test config to specify a timezone to use, but I can't find anything about it anymore. Did we ever figure out this problem or am I just dreaming?

Reply:
There was this commit where the change in `package.json` set Jest to run in UTC: https://github.com/lyft/amundsenfrontendlibrary/commit/b3ed6288bcc0dce5e1564d5bfe7ccc3f2aef927e#diff-fc699096242ec6e9acf55ca6bf7ecf09